### DISCUSSION TITLE
How can I use an component attribute in a custom form field ?

### DISCUSSION THREAD

**Discussion by romaldyminaya** [filamentphp/filament#1714] [General]
Hi,

I have this component:

```
use Closure;
use Filament\Forms\Components\Field;

class Timeline extends Field
{
    protected string $view = 'forms.components.time-line';
    protected string | Closure | null $relationship = 'timeline';

    public $comment = 'Hello world';
}
```

But in the view i'm not able to sync the attribue `$comment`

```
<x-forms::field-wrapper
    :id="$getId()"
    :label="null"
    :label-sr-only="$isLabelHidden()"
    :helper-text="$getHelperText()"
    :hint="$getHint()"
    :hint-icon="$getHintIcon()"
    :required="false"
    :state-path="$getStatePath()"
>
    <input wire:model="comment" type="text" />
</x-forms::field-wrapper>
```
How can I achieve this?

**Comment by danharrin** [comment]
Fields are not Livewire components, you cant just bind to public properties like that. What are you trying to do?

**Reply by romaldyminaya** [reply]
Hi @danharrin I need to display a timeline in a resource form. The thing is that the timeline has its own attributes and methods to interact with. So I need to know how can I achieve this. How to bind custom properties and custom methods to a custom Form Field. Is there any way possible ?
